Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM node:4
+
+ENV LOUNGE_VERSION 1.5.0
+ENV NODE_ENV production
+
+ARG user=lounge
+ARG group=lounge
+ARG uid=1000
+ARG gid=1000
+
+ENV LOUNGE_HOME "/home/lounge"
+ENV LOUNGE_DATA "${LOUNGE_HOME}/data"
+ENV LOUNGE_SRC "${LOUNGE_HOME}/src"
+
+# Create a non-root user for lounge to run in.
+RUN groupadd --gid ${gid} ${group} \
+      && useradd --home "${LOUNGE_HOME}" --create-home --uid ${uid} --gid ${gid} ${user}
+
+VOLUME "${LOUNGE_DATA}"
+RUN chown -R ${user} "${LOUNGE_DATA}"
+
+# Drop root.
+USER ${user}
+
+# Install thelounge.
+RUN mkdir -p "${LOUNGE_SRC}"
+WORKDIR "${LOUNGE_SRC}"
+RUN curl -L $(npm view thelounge@${LOUNGE_VERSION} dist.tarball) | tar --strip-components 1 -xzf -
```

Comment:
I like to think that, conceptually, Docker and npm are on the same level on the distribution layer, which makes it a bit weird to bundle the lounge image with a global npm installation of thelounge. In terms of distributing containerized software through Docker, I think the current way makes the most sense.